### DISCUSSION TITLE
Shortcut for openning settings

### DISCUSSION THREAD

**Discussion by tommdq** [zen-browser/desktop#3786] [Q&A]
I've been using Zen for a few months and haven't found a way to add a keyboard shortcut for opening settings. Is there a workaround to do this?

**Comment by llricci** [comment]
I don't think so. I think it would be helpful to suggest adding a shortcut for that. i.e.: `Ctrl/Command + ,`

**Comment by hoangdinhini** [comment]
Ctrl + , would be great. I found some app on Windows adopting this behavior from Macos.
